app/main.py

```diff
@@ -1,3 +1,36 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import os
+from decimal import Decimal
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open(f"app{os.sep}config.json", "r") as file:
+        trip_data = json.load(file)
+
+    customers, shops = [], []
+    fuel_price = Decimal(str(trip_data["FUEL_PRICE"]))
+
+    for cust_characteristics in trip_data["customers"]:
```

Comment:
Three for loops over the same data is a bit too much. You can reduce it to two loops for efficeincy. I would recommend to create Shops first, and then iterate over same data set to create customers, cars and do other things